pkgs/by-name/sl/sleek-todo/package.nix

```
@@ -68,6 +68,9 @@ else
       ;
 
```

```suggestion
	nativeBuildInputs = [ makeWrapper ];
```

The setup hook will be called automatically when placed in `nativeBuildInputs`.